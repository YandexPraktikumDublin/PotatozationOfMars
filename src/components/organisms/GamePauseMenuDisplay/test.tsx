import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { GamePauseMenuDisplay } from '.'

describe('<GamePauseMenuDisplay />', () => {
  it('should renders correct <GamePauseMenuDisplay />', () => {
    const toggleModal = jest.fn()
    const isGamePaused = true

    const wrapper = shallow(
      <GamePauseMenuDisplay
        isGamePaused={isGamePaused}
        toggleModal={toggleModal}
      />
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
