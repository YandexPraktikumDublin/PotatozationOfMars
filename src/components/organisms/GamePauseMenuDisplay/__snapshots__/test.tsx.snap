// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GamePauseMenuDisplay /> should renders correct <GamePauseMenuDisplay /> 1`] = `
<Fragment>
  <GamePauseMenuModal
    toggleModal={[MockFunction]}
  />
</Fragment>
`;
